portiaui/app/styles/templates/browsers.scss

```diff
@@ -0,0 +1,18 @@
+.browser-logos {
+    height: 120px;
+}
+
+.browser-logo {
+    height: 120px;
```

Comment:
lets make these images size based on the container width so that they always fix next to each other on all screen sizes, and set this as a max-height